code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -440,10 +440,14 @@
 		to_chat(src, "<span class='notice'>[M] shakes you to get you up!</span>")
 
 	else if(check_zone(M.zone_selected) == BODY_ZONE_HEAD) //Headpats!
+		SEND_SIGNAL(src, COMSIG_CARBON_HEADPAT, M)
 		M.visible_message("<span class='notice'>[M] gives [src] a pat on the head to make [p_them()] feel better!</span>", \
-					"<span class='notice'>You give [src] a pat on the head to make [p_them()] feel better!</span>")
+					null, "<span class='hear'>You hear a soft patter.</span>", DEFAULT_MESSAGE_RANGE, list(M, src))
+		to_chat(M, "<span class='notice'>You give [src] a pat on the head to make [p_them()] feel better!</span>")
+		to_chat(src, "<span class='notice'>[M] gives you a pat on the head to make you feel better! </span>")
```

Comment:
what

Reply:
This is a side-fix for blind people being unable to feel headpats, since I was already giving it a signal(I changed it to how hugs display their message). I can remove it if you want